treeherder/perf/stats.py

```diff
@@ -339,33 +339,37 @@ def is_new_better(is_effect_meaningful, is_base_greater, is_significant, lower_i
     return direction, is_new_better
 
 
+# 0 signifies no change in shift between confidence interval range
 def interpret_performance_direction(ci_low, ci_high, lower_is_better):
     is_regression = False
     is_improvement = False
-    ci_intepretation = None
+    ci_interpretation = "Failed to compute. Could be needing more data."
     if ci_high is None or ci_low is None:
-        return None, None, None
-    if ci_low > 0:
-        if lower_is_better:
+        return None, None, ci_interpretation
+
+    if lower_is_better:
+        # the upper bound of the range shift < 0 indicates strong likelyhood of a negative shift
+        if ci_high <= 0:
             is_regression = False
             is_improvement = True
-            ci_intepretation = "Performance improved (median increased)"
-        else:
+            ci_interpretation = "Performance likely improved (median decreased)"
+        # the lower bound of the range shift > 0 indicates strong likelyhood of a positive shift
+        if ci_low >= 0:
             is_regression = True
             is_improvement = False
-            ci_intepretation = "Performance regressed (median increased)"
-    elif ci_high < 0:
-        if lower_is_better:
```

Comment:
If we find we just need to flip the result, could we just change the `if lower_is_better` to `if not lower_is_better`?